bedrock/pocket/templates/pocket/home.html

```diff
@@ -0,0 +1,119 @@
+          {% endcall %}
+
+        <section>
+          {% call split(
+          block_class='mzp-l-split-center-on-sm-md mzp-l-split-reversed',
+          media_class='mzp-l-split-h-center',
+          media_after=True,
+          image_url='img/pocket/pocket-colorful-books.svg',
+          )%}
+          <h2 class="sub-section-heading">{{ ftl('pocket-home-save-interesting-stories') }}</h2>
+          <p class="section-lede">{{ ftl('pocket-home-stop-sending-yourself-links') }}</p>
+          {% endcall %}
+        </section>
+
+        <section>
+          {% call split(
+          block_class='mzp-l-split-center-on-sm-md',
+          media_class='mzp-l-split-h-center',
+          media_after=True,
+          image_url='img/pocket/pocket-list-mobile.png',
+          )%}
+          <h2 class="sub-section-heading">{{ ftl('pocket-home-great-recommendations') }}</h2>
+          <p class="section-lede">{{ ftl('pocket-home-we-comb-the-internet') }}</p>
+          {% endcall %}
+        </section>
+
+        <section class="pocket-homepage-picto mzp-l-content mzp-t-content-xl">
+
+          <h2 class="sub-section-heading">{{ ftl('pocket-home-make-pocket-work') }}</h2>
+           <ul class="mzp-l-columns mzp-t-columns-three">
+            {% call picto(
+              title=ftl('pocket-home-tailor-text'),
+              image_url='img/pocket/pocket-text-icon.svg',
+              image_width=40
+            ) %}
+            {% endcall %}
+
+            {% call picto(
+              title=ftl('pocket-home-categorize-saves'),
+              image_url='img/pocket/pocket-tag-icon.svg',
+              image_width=40
+            ) %}
+            {% endcall %}
+
+            {% call picto(
+              title=ftl('pocket-home-listen-to-articles'),
+              image_url='img/pocket/pocket-audio-icon.svg',
+              image_width=40,
+            ) %}
+            {% endcall %}
+
+            </ul>
+        </section>
+
+        <section class="pocket-homepage-premium ">
+          <div class="mzp-l-content mzp-t-content-xl">
+            <div class="mzp-l-columns mzp-t-columns-two">
+              <div>
+                <h2>{{ ftl('pocket-home-start-with-the-best') }}</h2>
+                <p>{{ ftl('pocket-home-pocket-premium-is') }}</p>
+              </div>
+              <div>
+                <ul class="mzp-u-list-styled">
+                  <li>{{ ftl('pocket-home-permanent-library') }}</li>
+                  <li>{{ ftl('pocket-home-suggested-tags') }}</li>
+                  <li>{{ ftl('pocket-home-full-text-search') }}</li>
+                  <li>{{ ftl('pocket-home-unlimited-highlights') }}</li>
+                  <li>{{ ftl('pocket-home-premium-fonts') }}</li>
+                </ul>
+                <a href="https://getpocket.com/premium#plans" class="mzp-c-button" target="_blank" rel="external noopener">Get Pocket Premium</a>
```

Comment:
Question: Is there a reason why we're adding `target="_blank" rel="external noopener"` to these links? They would be same-site (right?). Do we need to treat them like an external domain? If not, I'd vote for removing these on getpocket.com links.

Reply:
This is a good point. In my mind, they were external because they were pages marketing did not control, but that likely doesn't matter. +1 for removing these attributes

Reply:
Good point. I'll also scour other Pocket files for this change.